src/tests/test_verify_schema.sh

```diff
@@ -48,7 +69,9 @@ for DATA_TYPE in miscproject vosummary rgsummary; do
     # Resource group and VO readers should use the contact info if we have
     # access to the SSH keys for the contacts repo
     if [[ $DATA_TYPE == 'vosummary' ]] || [[ $DATA_TYPE == 'rgsummary' ]]; then
-        if [[ "$TRAVIS_PULL_REQUEST" == "false" ]]; then
+        if [[ $TRAVIS_PULL_REQUEST == "false" ||
+              $GH_EVENT == 'push' &&
+              $GITHUB_REPOSITORY == 'opensciencegrid/topology' ]]; then
```

Comment:
Same here, if this is what you meant

```suggestion
        if [[ $TRAVIS_PULL_REQUEST == "false" || $GH_EVENT == 'push' ]] &&
           [[ $GITHUB_REPOSITORY == 'opensciencegrid/topology' ]]; then
```